web/libs/editor/src/components/SidePanels/DetailsPanel/Relations.tsx

```diff
@@ -10,9 +10,13 @@ import { Select } from 'antd';
 import './Relations.styl';
 
 const RealtionsComponent: FC<any> = ({ relationStore }) => {
+  const relations = useCallback(() => {
+    return relationStore.orderedRelations;
+  }, [relationStore])();
```

Comment:
```suggestion
  const relations = relationStore.orderedRelations;
```
I'm not sure what the use case was for using a useCallback like a useMemo, but from what I can see this would be more likely the scenario that would be expected here is to allow the reactive property to be passed through directly.